front/lib/models/assistant/actions/retrieval.ts

```diff
@@ -185,18 +192,32 @@ AgentDataSourceConfiguration.init(
 AgentRetrievalConfiguration.hasOne(AgentConfiguration, {
   foreignKey: { name: "retrievalConfigurationId", allowNull: true }, // null = no retrieval action set for this Agent
 });
+AgentConfiguration.belongsTo(AgentRetrievalConfiguration, {
```

Comment:
Why do we need this?

Reply:
Same as before the `as` can be added in the `hasOne` and this one can be removed